file-formats/cfdf/type/zif_aff_cfdf_v1.intf.abap

```diff
@@ -0,0 +1,592 @@
+      "! E-Mail
+      email                      TYPE ty_field_type VALUE 'EMAIL' ##NO_TEXT,
+      "! <p class="shorttext">List</p>
+      "! List
+      list                       TYPE ty_field_type VALUE 'LIST' ##NO_TEXT,
+      "! <p class="shorttext">Number</p>
+      "! Number
+      number                     TYPE ty_field_type VALUE 'NUMBER' ##NO_TEXT,
+      "! <p class="shorttext">Quantity with Unit</p>
+      "! Quantity with unit
+      quantity                   TYPE ty_field_type VALUE 'QUANTITY' ##NO_TEXT,
+      "! <p class="shorttext">Text</p>
+      "! Text
+      text                       TYPE ty_field_type VALUE 'TEXT' ##NO_TEXT,
+      "! <p class="shorttext">Time</p>
+      "! Time
+      time                       TYPE ty_field_type VALUE 'TIME' ##NO_TEXT,
+      "! <p class="shorttext">Web Address</p>
+      "! Web address
+      url                        TYPE ty_field_type VALUE 'URL' ##NO_TEXT,
+      "! <p class="shorttext">Phone Number</p>
+      "! Phone number
+      phone                      TYPE ty_field_type VALUE 'PHONE' ##NO_TEXT,
+      "! <p class="shorttext">Amount Only</p>
+      "! Amount only
+      amount_only                TYPE ty_field_type VALUE 'AMNT_ONLY' ##NO_TEXT,
+      "! <p class="shorttext">Currency</p>
+      "! Currency
+      currency_code              TYPE ty_field_type VALUE 'CURRENCY' ##NO_TEXT,
+      "! <p class="shorttext">Quantity Only</p>
+      "! Quantity only
+      quantity_only              TYPE ty_field_type VALUE 'QUAN_ONLY' ##NO_TEXT,
+      "! <p class="shorttext">Unit</p>
+      "! Unit
+      unit_code                  TYPE ty_field_type VALUE 'UNIT' ##NO_TEXT,
+      "! <p class="shorttext">Language</p>
+      "! Language
+      language_code              TYPE ty_field_type VALUE 'LANGUAGE' ##NO_TEXT,
+      "! <p class="shorttext">Numerical Text</p>
+      "! Numerical text
+      numerical_text             TYPE ty_field_type VALUE 'NUMC_TEXT' ##NO_TEXT,
+      "! <p class="shorttext">Posting Perioid</p>
+      "! Posting perioid
+      posting_period             TYPE ty_field_type VALUE 'PST_PERIOD' ##NO_TEXT,
+      "! <p class="shorttext">Raw</p>
+      "! Raw
+      raw                        TYPE ty_field_type VALUE 'RAW' ##NO_TEXT,
+      "! <p class="shorttext">Association to Standard Business Object</p>
+      "! Association to standard business object
+      association_to_standard_bo TYPE ty_field_type VALUE 'ASC_TO_STD' ##NO_TEXT,
+      "! <p class="shorttext">Timestamp</p>
+      "! Timestamp
+      timestamp                  TYPE ty_field_type VALUE 'TIMESTAMP' ##NO_TEXT,
+      "! <p class="shorttext">Code List based on CDS View</p>
+      "! Code list based on CDS view
+      list_cview                 TYPE ty_field_type VALUE 'LIST_CVIEW' ##NO_TEXT,
+      "! <p class="shorttext">Characteristics - List</p>
+      "!Characteristics - List
+      characteristic_list        TYPE ty_field_type VALUE 'CHRCS_LIST' ##NO_TEXT,
```

Comment:
characteristics_list?

Reply:
I agree, characteristics_list is a better name. A custom field of type characteristcs_list is a wrapper of an characteristics object (details https://fioriappslibrary.hana.ondemand.com/sap/fix/externalViewer/#/detail/Apps('F4337')/S29)